doc/sphinx-guides/source/installation/oidc.rst

```diff
@@ -80,9 +123,64 @@ The Dataverse installation will automatically try to load the provider and retri
 You should see the new provider under "Other options" on the Log In page, as described in the :doc:`/user/account`
 section of the User Guide.
 
-By default, the Log In page will show the "builtin" provider, but you can adjust this via the ``:DefaultAuthProvider``
-configuration option. For details, see :doc:`config`.
-
-.. hint::
-   In contrast to our :doc:`oauth2`, you can use multiple providers by creating distinct configurations enabled by
-   the same technology and without modifying the Dataverse Software code base (standards for the win!).
+.. _oidc-mpconfig:
+
+Provision via MPCONFIG
+^^^^^^^^^^^^^^^^^^^^^^
+
+In case you only require a single OIDC provider, you can also provision an OIDC based provider using MicroProfile Config
```

Comment:
This looks great but I'm a little concerned that a new user will be confused about MPCONFIG and how to configure it. Either in this PR or a future one, maybe we can link to a blurb about this (on the main config.rst page, I assume).

Reply:
Done with a2b9d56f